### DISCUSSION TITLE
Running supabase on the Red Hat Linux Server

### DISCUSSION THREAD

**Discussion by raman04-byte** [supabase/supabase#28349] [Questions]
Well I am trying to run the supabase localhost instance to the server but the analytics container is failing to load

![image](https://github.com/user-attachments/assets/58e064a5-5b76-4e00-ad20-95409b9ded22)

**Comment by GaryAustin1** [comment]
Make sure you have the latest CLI version.
You should search these discussions and supabase/cli issues.
This is one mention but there were a few others:  https://github.com/supabase/cli/issues/2503

If those don't help, I'm not going to be the one to help and you'll have to see if someone else comes along.

**Reply by raman04-byte** [reply]
I am not using CLI for this just by cloning the repo and building it.
